packages/ag-charts-community/src/chart/axis/axis.ts

```diff
@@ -1145,14 +1143,21 @@ export abstract class Axis<S extends Scale<D, number, TickInterval<S>> = Scale<a
     }
 
     protected calculateAvailableRange(): number {
-        const { range: requestedRange } = this;
+        const { range } = this;
 
-        const min = Math.min(...requestedRange);
-        const max = Math.max(...requestedRange);
+        const min = Math.min(...range);
+        const max = Math.max(...range);
 
         return max - min;
     }
 
+    protected calculateVisibleRange() {
+        const { visibleRange } = this;
+        const visibleScale = 1 / (visibleRange[1] - visibleRange[0]);
+
+        return this.calculateAvailableRange() * visibleScale;
+    }
```

Comment:
The `calculateAvailableRange()` function is still used for the label collision checking